doc/lightning-listpeers.7.md

```diff
@@ -74,8 +74,8 @@ On success, an object containing **peers** is returned.  It is an array of objec
   - **funding** (object, optional):
     - **local\_funds\_msat** (msat): Amount of channel we funded
     - **remote\_funds\_msat** (msat): Amount of channel they funded
-    - **local\_msat** (msat, optional): Amount of channel we funded (deprecated)
-    - **remote\_msat** (msat, optional): Amount of channel they funded (deprecated)
+    - **local\_msat** (msat, optional): Amount of channel we funded **deprecated, removal in v23.5**
+    - **remote\_msat** (msat, optional): Amount of channel they funded **deprecated, removal in v23.5**
```

Comment:
Needs `0`-padding on the month, otherwise lexicographic sorting of versions is broken.